plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -944,306 +1061,371 @@ void producer_plugin::plugin_initialize(const boost::program_options::variables_
-           ("ww", my->_ro_write_window_time_us)("rw", my->_ro_read_window_time_us)("w", my->_ro_read_window_effective_time_us));
+      ilog("read-only-write-window-time-us: ${ww} us, read-only-read-window-time-us: ${rw} us, effective read window time to be used: ${w} "
+           "us",
+           ("ww", _ro_write_window_time_us)("rw", _ro_read_window_time_us)("w", _ro_read_window_effective_time_us));
    }
 
    // Make sure _ro_max_trx_time_us is alwasys set.
-   if ( my->_max_transaction_time_ms.load() > 0 ) {
-      my->_ro_max_trx_time_us = fc::milliseconds(my->_max_transaction_time_ms.load());
+   if (_max_transaction_time_ms.load() > 0) {
+      _ro_max_trx_time_us = fc::milliseconds(_max_transaction_time_ms.load());
    } else {
       // max-transaction-time can be set to negative for unlimited time
-     my->_ro_max_trx_time_us = fc::microseconds::maximum();
+      _ro_max_trx_time_us = fc::microseconds::maximum();
    }
-   ilog("read-only-threads ${s}, max read-only trx time to be enforced: ${t} us", ("s", my->_ro_thread_pool_size)("t", my->_ro_max_trx_time_us));
+   ilog("read-only-threads ${s}, max read-only trx time to be enforced: ${t} us", ("s", _ro_thread_pool_size)("t", _ro_max_trx_time_us));
 
-   my->_incoming_block_sync_provider = app().get_method<incoming::methods::block_sync>().register_provider(
-         [this](const signed_block_ptr& block, const std::optional<block_id_type>& block_id, const block_state_ptr& bsp) {
-      return my->on_incoming_block(block, block_id, bsp);
-   });
+   _incoming_block_sync_provider = app().get_method<incoming::methods::block_sync>().register_provider(
+      [this](const signed_block_ptr& block, const std::optional<block_id_type>& block_id, const block_state_ptr& bsp) {
+         return on_incoming_block(block, block_id, bsp);
+      });
 
-   my->_incoming_transaction_async_provider = app().get_method<incoming::methods::transaction_async>().register_provider(
-         [this](const packed_transaction_ptr& trx, bool api_trx, transaction_metadata::trx_type trx_type, bool return_failure_traces, next_function<transaction_trace_ptr> next) -> void {
-      return my->on_incoming_transaction_async(trx, api_trx, trx_type, return_failure_traces, next );
-   });
+   _incoming_transaction_async_provider =
+      app().get_method<incoming::methods::transaction_async>().register_provider([this](const packed_transaction_ptr&  trx,
+                                                                                        bool                           api_trx,
+                                                                                        transaction_metadata::trx_type trx_type,
+                                                                                        bool return_failure_traces,
+                                                                                        next_function<transaction_trace_ptr> next) -> void {
```

Comment:
Why did it line up `trx`, `api_trx`, `trx_type`, but not `return_failure_traces`. I prefer just single spaces between types and params.

Reply:
yes, fixed.